section_faiss_vector_index.go

```diff
@@ -591,11 +531,9 @@ func (vo *vectorIndexOpaque) process(field index.VectorField, fieldID uint16, do
 		subVecHash := hashCode(subVec)
 		if _, ok := vo.vecIDMap[subVecHash]; !ok {
 			vo.vecIDMap[subVecHash] = vecInfo{
-				docIDs: roaring.NewBitmap(),
+				docIDs: uint64(docNum),
```

Comment:
Why cast a docNum to uint64 here, leave it at uint32. docIDs needs refactoring to docID with your proposal.

Reply:
Did that to avoid casting here - https://github.com/blevesearch/zapx/blob/511fd66ddea9a1301b99f240d42fadb0c3ee7e16/section_faiss_vector_index.go#L495